src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="search"
+    method="get"
+    data-qa="big"
+    class="form"
+  >
```

Comment:
```suggestion
    <form
      action="search"
      method="get"
      data-qa="big"
      class="form"
    >
```
Change inden
![Screenshot 2022-07-19 at 06 49 45](https://user-images.githubusercontent.com/17098736/179660583-7efc9f19-1cd5-4664-a740-437c3619427c.png)
ts.